### PR TITLE
add tests through gh actions

### PR DESCRIPTION
**Pull request by ppanero**

### CODE REVIEW

File: run-tests.sh [hunk 1]
```diff
@@ -12,8 +12,13 @@
...
 # released.
 
-docker-services-cli up postgresql es redis
-python -m check_manifest --ignore ".travis-*" && \
+# Quit on errors
+set -o errexit
+
+# Quit on unbound symbols
+set -o nounset
+
```
Comment (fenekku) [inline run-tests.sh:20]: Could `set -e` be enough? Why was this added now?

**Review by fenekku — APPROVED** [review]
Just wandering about those `set` calls... :thinking:

File: run-tests.sh (hunk 1 above)
Comment (ppanero) [reply run-tests.sh:20]: `set -e` would replace the first, `set -u` would replace the second. I was just trying to follow what was agreed on expanding command-line arguments to be more explicit.

Why? Because it was not enough as it was. For example, if the manifest check was wrong it would continue to run tests. We want it to fail fast and stop in the first command that errors.

Comment (fenekku) [reply run-tests.sh:20]: ok!
